integrationtest/java/modules/pojo-standalone-lucene/src/main/java/org/hibernate/search/integrationtest/java/modules/pojo/standalone/lucene/service/AuthorService.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.java.modules.pojo.standalone.lucene.service;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.hibernate.search.engine.backend.work.execution.DocumentRefreshStrategy;
+import org.hibernate.search.engine.search.query.SearchQuery;
+import org.hibernate.search.integrationtest.java.modules.pojo.standalone.lucene.entity.Author;
+import org.hibernate.search.mapper.pojo.standalone.loading.SelectionLoadingStrategy;
+import org.hibernate.search.mapper.pojo.standalone.mapping.CloseableSearchMapping;
+import org.hibernate.search.mapper.pojo.standalone.mapping.SearchMapping;
+import org.hibernate.search.mapper.pojo.standalone.session.SearchSession;
+
+public class AuthorService implements AutoCloseable {
+
+	private static final AtomicInteger ID_PROVIDER = new AtomicInteger( 1 );
```

Comment:
Good idea; while not strictly necessary in the context of these tests, it's indeed cleaner than hardcoding the ID as I was suggesting, due to how the tests are separate from the creation of entities (I overlooked that part).